samples/Azure.ResourceManager.Sample/Generated/AvailabilitySetContainer.cs

```diff
@@ -121,7 +121,7 @@ public AvailabilitySetsCreateOrUpdateOperation StartCreateOrUpdate(string availa
                 }
 
                 var originalResponse = _restClient.CreateOrUpdate(Id.ResourceGroupName, availabilitySetName, parameters, cancellationToken: cancellationToken);
-                return new AvailabilitySetsCreateOrUpdateOperation(Parent, originalResponse);
+                return new AvailabilitySetsCreateOrUpdateOperation(this, originalResponse);
```

Comment:
Is this expected?

Reply:
Originally, I wanted to avoid using the Parent as much as possible. But it will other issues to use `this` here. I reverted the change.